sdk/messaging/azeventgrid/go.mod

```diff
@@ -1,3 +1,5 @@
+// Deprecated: Event Grid Basic client has moved to github.com/Azure/azure-sdk-for-go/sdk/messaging/eventgrid/azeventgrid
+// Deprecated: Event Grid Namespaces client has moved to github.com/Azure/azure-sdk-for-go/sdk/messaging/eventgrid/aznamespaces
```

Comment:
Do we want the second `Deprecated:`? The [docs](https://go.dev/ref/mod#go-mod-file-module-deprecation) imply it's redundant and would appear as ordinary text in the message, but I dunno because I haven't seen a published case like this.

Reply:
Hm.

> A module can be marked as deprecated in a block of comments containing the string Deprecated: (case-sensitive) at the beginning of a paragraph.

Now I'm wondering what that "a block of comments" actually means. We do need to list both modules, but I could list them in one long sentence to be safe. I will check.

Reply:
Updated to just avoid the discussion :)